.github/workflows/ci.yml

```diff
@@ -19,13 +18,22 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: ${{ matrix.go }}
+      - name: Check out code the Go-Way
+        run: go get github.com/${GITHUB_REPOSITORY}
       - name: Setup
         run: |
+          cd ~/go/src/github.com/${GITHUB_REPOSITORY}
+          go get -v -t -d ./...
```

Comment:
This isn't necessary as there are no modules outside the core library that this project uses.

Reply:
Please see https://github.com/mattn/goveralls/issues/140 but yes I agree, was just hacking this together to get it to work initially.